decentralized_devstack/provision-lms.sh

```diff
@@ -3,11 +3,18 @@
 # Include utilities.
 source decentralized_devstack/provisioning-utils.sh
 
-log_step "lms: Ensuring MySQL databases and users exist..."
-docker-compose exec -T mysql bash -c "mysql -uroot mysql" < decentralized_devstack/provision-mysql-lms.sql
-
-log_step "lms: Adding default MySQL data from dump..."
-docker-compose exec -T mysql /usr/bin/mysql edxapp < decentralized_devstack/provision-mysql-lms-data.sql
+# only load mysql schema if database does not exist, this way reprovisioning will not overright data
+edxapp_count=$(docker-compose exec mysql mysql -uroot --skip-column-names -se "SELECT COUNT(*) FROM INFORMATION_SCHEMA.SCHEMATA WHERE SCHEMA_NAME='edxapp';" 2>/dev/null | tr -d $'\r')
+if [[ "${edxapp_count}" = "1" ]];
+then
+log_step "LMS DB exists, skipping lms schema load.."
```

Comment:
```suggestion
if [[ "${edxapp_count}" = "1" ]]; then
  log_step "LMS DB exists, skipping lms schema load.."
```

Just a formatting suggestion.